### PR TITLE
created sign up and log in page

### PR DESCRIPTION
**Pull request by ajoxales**
created sign up and log in page. redirects to different portal based on user's role type.

### CODE REVIEW

File: src/app/assets/signup.jpg
Comment (jaymercado) [inline src/app/assets/signup.jpg:1]: remove this file. You already have this file inside the public folder.

File: src/app/assets/signup2.webp
Comment (jaymercado) [inline src/app/assets/signup2.webp:1]: Do you need this file? If yes, move it inside the public folder.

File: src/app/assets/style.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+* {    
+    color: #0e0e0e;
+    font-size: 1rem;
+    letter-spacing: -0.4px;
+
```
Comment (jaymercado) [inline src/app/assets/style.css]: Remove extra space (lines 5 and 6)

File: src/app/assets/style.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    
+}
+
+h1 {
```
Comment (jaymercado) [inline src/app/assets/style.css]: remove this and use styled-components

File: src/app/layout.js [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
-
-const inter = Inter({ subsets: ['latin'] })
+import { Poppins } from "next/font/google";
+import style from "@/app/assets/style.css";
+const poppins = Poppins({
+  subsets: ["latin"],
+  weight: ["100", "200", "300", "400", "500", "600", "700", "800", "900"],
+});
 
 export const metadata = {
```
Comment (jaymercado) [inline src/app/layout.js:8]: Do you need this metadata? If not, please remove this

File: src/app/login/assets/style.css
Comment (jaymercado) [inline src/app/login/assets/style.css:1]: Remove this file and use styled-components

File: src/app/login/page.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+      if (value === "") {
+        incompleteInput = true;
+      }
+    });
+    if (!incompleteInput) setIsValid(false);
+    else setIsValid(true);
+  }, [values]);
+
+  return (
+    <div className="d-flex justify-content-center align-items-center">
```
Comment (jaygmercado) [inline src/app/login/page.js:59]: No inline-styling. Use styled-components

File: src/app/login/page.js [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+        <div className="d-flex">
+          <p>Already have an account?</p>
+          <span className="ms-2 fw-medium">
+            <Link href="/signup">Sign Up</Link>
+          </span>
+        </div>
+        <Button variant="primary" disabled={isValid} onClick={login}>
+          Login
+        </Button>
+        <span style={{ color: "red" }}>{error}</span>
```
Comment (jaymercado) [inline src/app/login/page.js]: No inline-styling. Use styled-components.

File: src/app/login/page.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+      <Form
+        onSubmit={(e) => {
+          e.preventDefault();
+        }}
+        className="form rounded bg-white">
+        <header className="mb-3">
+          <h1 className="sign-up fw-bold">Log In</h1>
+          <h2 className="subtitle">Login to your account</h2>
+        </header>
+        <Form.Group className="mb-3" controlId="exampleForm.ControlInput2">
```
Comment (jaymercado) [inline src/app/login/page.js]: remove exampleForm inside controlId. Use a better name like usernameInput

File: src/app/login/page.js [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+          <Form.Label className="label">Username</Form.Label>
+          <Form.Control
+            type="text"
+            name="username"
+            placeholder="juandelacruz"
+            value={values.username}
+            onChange={handleChange}
+          />
+        </Form.Group>
+        <Form.Group className="mb-3" controlId="exampleForm.ControlInput4">
```
Comment (jaymercado) [inline src/app/login/page.js]: remove exampleForm inside controlId. Use a better name like passwordInput

File: src/app/signup/assets/style.css
Comment (jaymercado) [inline src/app/signup/assets/style.css:1]: Delete file and use styled-components.

File: src/app/signup/page.js [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+          e.preventDefault();
+        }}
+        className="form rounded bg-white">
+        <header>
+          <h1 className="sign-up fw-bold">Sign Up</h1>
+          <h2 className="subtitle">Create your account</h2>
+        </header>
+        <div className="row mt-4">
+          <Form.Group
+            className="col mb-3"
```
Comment (jaymercado) [inline src/app/signup/page.js]: use Col from react/bootstrap instead of col class.

File: src/app/signup/page.js [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+            controlId="exampleForm.ControlInput1">
```
Comment (jaymercado) [inline src/app/signup/page.js]: remove exampleForm inside controlId. Use a better name like nameInput.

File: src/app/signup/page.js [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+            <Form.Control
+              type="text"
+              name="firstName"
+              placeholder="Juan"
+              value={values.firstName}
+              onChange={handleChange}
+            />
+          </Form.Group>
+          <Form.Group
+            className="col mb-3"
```
Comment (jaymercado) [inline src/app/signup/page.js]: use Col from react/bootstrap instead of col class.

File: src/app/signup/page.js [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+            controlId="exampleForm.ControlInput2">
```
Comment (jaymercado) [inline src/app/signup/page.js]: remove exampleForm inside controlId. Use a better name like lastNameInput.

File: src/app/signup/page.js [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+            <Form.Control
+              type="text"
+              name="lastName"
+              placeholder="Dela Cruz"
+              value={values.lastName}
+              onChange={handleChange}
+            />
+          </Form.Group>
+        </div>
+        <Form.Group className="mb-3" controlId="exampleForm.ControlInput2">
```
Comment (jaymercado) [inline src/app/signup/page.js]: remove exampleForm inside controlId. Use a better name like usernameInput.

File: src/app/signup/page.js [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+          <Form.Label className="label">Username</Form.Label>
+          <Form.Control
+            type="text"
+            name="username"
+            placeholder="juandelacruz"
+            value={values.username}
+            onChange={handleChange}
+          />
+        </Form.Group>
+        <Form.Group className="mb-3" controlId="exampleForm.ControlInput3">
```
Comment (jaymercado) [inline src/app/signup/page.js]: remove exampleForm inside controlId. Use a better name like emailInput.

File: src/app/signup/page.js [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+          <Form.Label className="label">Email address</Form.Label>
+          <Form.Control
+            type="email"
+            name="emailAddress"
+            placeholder="juandelacruz@gmail.com"
+            value={values.emailAddress}
+            onChange={handleChange}
+          />
+        </Form.Group>
+        <Form.Group className="mb-3" controlId="exampleForm.ControlInput4">
```
Comment (jaymercado) [inline src/app/signup/page.js]: remove exampleForm inside controlId. Use a better name like passwordInput.

File: src/app/signup/page.js [hunk 15]
```diff
@@ -0,0 +1,147 @@
...
+        <div className="d-flex">
+          <p>Already have an account?</p>
+          <span className="ms-2 fw-medium">
+            <Link href="/login">Login</Link>
+          </span>
+        </div>
+        <Button variant="primary" disabled={isValid} onClick={signup}>
+          Sign Up
+        </Button>
+        <span style={{ color: "red" }}>{error}</span>
```
Comment (jaymercado) [inline src/app/signup/page.js]: No inline styling. Use styled-components

**Pull request merged**